payu/runlog.py

```diff
@@ -95,9 +96,18 @@ def commit(self):
         print(cmd)
         try:
             subprocess.check_call(shlex.split(cmd), stdout=f_null,
-                                  cwd=self.expt.control_path)
+                                  stderr=f_null, cwd=self.expt.control_path)
         except subprocess.CalledProcessError:
-            print('TODO: Check if commit is unchanged')
+            # Attempt commit without signing commits
+            cmd = f'git -c "commit.gpgsign=false" commit -am "{commit_msg}"'
```

Comment:
There is also a `--no-gpg-sign` flag for `git commit`, if you didn't want to modify config.

Reply:
Thanks! Yeah that makes the command a little more easier to read. I've added it in 55f594a and have tested that it works the same on a payu run job that errors with signing commits .